src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -85,6 +86,21 @@ public static Address parseAddress(String address) throws ParseException {
         return new Address(trimmedAddress);
     }
 
+    /**
+     * Parses a {@code String company} into an {@code Company}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code company} is invalid.
+     */
+    public static Company parseCompany(String company) throws ParseException {
+        requireNonNull(company);
+        String trimmedCompany = company.trim();
+        if (!Address.isValidAddress(trimmedCompany)) {
```

Comment:
Is this still using the oudated Address method and should it be replaced with the method in Company?